vocode/streaming/models/events.py

```diff
@@ -38,4 +39,17 @@ class RecordingEvent(Event, type=EventType.RECORDING):
 
 class ActionEvent(Event, type=EventType.ACTION):
     action_input: Optional[dict] = None
-    action_output: Optional[dict] = None
\ No newline at end of file
+    action_output: Optional[dict] = None
+
+
+class FillerEvent(Event, type=EventType.PUBLISH_FILLER):
+    filler_phrase: str
+
+
+class SpokenMetaData:
```

Comment:
nit: typically it would be `Metadata` with a lower `d`

also, not sure how useful the comment below is :)

Reply:
will fix and add more docstring